operations/onboarding.md

```diff
@@ -0,0 +1,99 @@
+* `:exclamation:` (red exclamation point) This emoji is used to indicate an action is needed by the TOC, either for an individual or for the group.
+
+Emojis can be used individually or with other emojis, this usually happens to indicate a reminder has an action with it or a reminder is informational but an update may make it more topical. It is common practice to **bold** the topic immediately following the emoji for longer updates, such as summarization of a meeting.
+
+ We operate on lazy consensus, if a TOC member is not participating in a discussion, it is their responsibility to keep themselves informed of the outcome and impetus of these discussion to guide future decision making as part of their participation in the TOC, the use of emojis and summarization of topics is intended to facillitate this in the most effective manner for a group of time-constrained individuals.
+
+### Due Diligence
+
+The primary work a TOC member engages is in conducting the due diligence of cloud native projects as they advance in their maturity, this process is commonly referred to the moving levels or maturation process. After a TOC member steps forward to sponsor a project, it is anticipated that a kick-off meeting is held to set expectations for what the process will be like, provide rough timelines, collect additional information and allow the project to ask any questions.  This kick-off is commonly conducted as a virtual meeting, not lasting more than an hour. Additionally, TOC members should anticipate a few hours of concentrated review, analysing the project, its repository(ies), and clarifying the content of the due diligence. Additional time is expected to be spent conducting and summarizing Adopter interviews to ascertain the current usage of the project within the industry.
+
+## Voting
+
+The TOC primarily votes in one of two mechanisms, gitvote or on the mailing list. It is our preference that voting occur with gitvote in the repository as this provides better transparency and is easier to verify and track vote execution. Voting conducted over the mailing list is performed by responding with a +1 Binding (+1B), or -1 Binding (-1). TOC members may abstain if they have a conflict of interest. Indication of abstention from voting for any other reason is discouraged and should not be confused with not yet voting as indication of abstention is active, not passive, and results in no vote being rendered by that individual. In the absence of information, we cannot make an informed decision, so it is imperative that we ask questions openly and provide answers with clarity for our peer TOC members, particularly on topics that may not be their strength. A vote passes with two-thirds vote of votes cast based on the charter (see [6(c)(viii)](https://www.cncf.io/about/charter)).
```

Comment:
I do not recall having a mailing list vote after introducing the gitvote. Will we have any issues that require voting that cannot be solved in a GH issue with gitvote?

Reply:
I dont believe so as our decision making that requires votes should be documented/recorded and public. Thinking through sandbox applications, we used to vote on the call which was recorded and @amye tallied/tracked votes. Since we've moved many of our processes to more transparent mediums (issues, PRs, etc.) that allows for gitvote, I do not believe we have any left that require facilitation of a vote outside of GitHub.

Reply:
Perhaps we can move it as a secondary vote option, perhaps for sensitive issues requiring a closed vote (but cannot think of any scenarios where that was required)- aka a backup voting solution.

Reply:
We could add a link to https://github.com/cncf/gitvote